#36798 [CUBRID] Fix deletion of comments/default values from columns

longhaseng52 opened the pull request:
Resolve the issue that prevents users from deleting comments and default values from table columns using the UI.

longhaseng52 committed:
#36798 Fix deletion of comments/default values from columns

ShadelessFox commented:
Thanks for the contribution! Passing the PR to our QA team now.

uslss committed:
Merge branch 'devel' into CommentTableColumn

uslss commented:
In case the comment was deleted - default value that was set before is also deleted


https://github.com/user-attachments/assets/f015086b-497f-4141-a109-73014b20e65c

Also if, e.g. default value was set before and then user set comment - default value is deleted

https://github.com/user-attachments/assets/2ae7bdfd-f393-4d01-8398-b2f4592f3094

longhaseng52 committed:
#36798 Fix deletion of comments/default values from columns

longhaseng52 commented:
@uslss
I have fixed this.

> In case the comment was deleted - default value that was set before is also deleted
> 
>  dbeaver_BSYu39KGWh.mp4 
> Also if, e.g. default value was set before and then user set comment - default value is deleted
> 
>  dbeaver_VpOu8fLnG3.mp4

uslss committed:
Merge branch 'devel' into CommentTableColumn

uslss commented:
verified
